From 388c2b59aa7f91f5eff81eab5bec5dfbd837d888 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Sun, 02 Jun 2024 16:32:53 +0300
Subject: [PATCH] [FSW-0] fs1: adjust for specs v1.7

In vlsi tag 1.7 vuart regs are moved to pmmu block.
The regs are mapped in EMB specs reg_map header.
In order to use the registers an include to the relevant
specs_h15 header is added

Change-Id: Id256258d0da0a37993c3d8b906ba4971a4281c8e
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/fs1/arc/common.c b/arch/fs1/arc/common.c
index 1bac7d5..03b941e 100644
--- a/arch/fs1/arc/common.c
+++ b/arch/fs1/arc/common.c
@@ -31,6 +31,7 @@
 #include <pcie_flr_regs.h>
 #include <tran_chk_lbw_regs.h>
 #include <tran_chk_lbw_masks.h>
+#include <pmmu_pcie_wrap_regs.h>
 #include <fs1_blocks_linux_driver.h>
 #include <fs1_security.h>
 
